src/components/MoveAdder.test.ts

```diff
@@ -0,0 +1,94 @@
+import { IMove } from "battlemovr";
+import { expect } from "chai";
+import { stubBlankGame } from "../fakes.test";
+import { IPokemon } from "./Battles";
+import { MoveAdder } from "./MoveAdder";
+
+describe("MoveAdder", () => {
+    it("adds a new move to a Pokemon's moveset", (): void => {
+        //Arrange
+        const { fsp } = stubBlankGame();
+        const pokemon: IPokemon = fsp.equations.newPokemon({
+            level: 5,
+            title: "SQUIRTLE".split(""),
+        });
+        const peck: IMove = {
+            title: "Peck",
+            remaining: 10,
+            uses: 10,
+        };
+
+        //Act
+        fsp.moveadder.addMove(pokemon, peck, 1);
+
+        //Assert
+        expect(pokemon.moves[1].title === peck.title);
```

Comment:
Fun fact: you'll get much better test descriptions on failures if you use the more verbose the Chai assertions.

Instead of:

```typescript
expect(a === b);
```

The error message becomes much more informative with:

```typescript
expect(a).to.be.equal(b);
```

Please switch to that.